int64 truck_id
float32 angle
float32 speed
